apis/nucleus/src/components/Header.jsx

```diff
@@ -94,7 +74,7 @@ function Header({ layout, sn, anchorEl, hovering, focusHandler, titleStyles = {}
   );
 
   return (
-    <StyledGrid ref={containerRef} item container wrap="nowrap" className={cls.join(' ')}>
+    <StyledGrid item container wrap="nowrap" className={classes.containerStyle}>
```

Comment:
This means we do not use `containerTitleStyle` any longer, so that should be removed.
However, it alters the styling slightly for charts - more room is given to the "chart area" (8px more in height). I think it's better and the right thing to do. The question is how much impact it would mean for customers though 🤔

Reply:
`containerTitleStyle` is still used in `StyledGrid` which is the wrapper of title and subtitle area. https://github.com/qlik-oss/nebula.js/blob/main/apis/nucleus/src/components/Header.jsx#L21